lib/tooling_invoker/setup_input_files.rb

```diff
@@ -0,0 +1,11 @@
+module ToolingInvoker
+  class SetupInputFiles
+    include Mandate
+
+    initialize_with :job
+
+    def call
+      FileUtils.cp_r(job.input_efs_dir, job.source_code_dir)
+    end
+  end
+end
```

Comment:
This will become much more complex as times goes on, so I've pulled it out into it's own command in prep for that.